examples/with-expo-typescript/README.md

```diff
@@ -33,9 +33,9 @@ Deploy the native app to the App store and Play store using [Expo deployment](ht
 Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
 
 ```bash
-npx create-next-app --example with-expo with-expo-app
+npx create-next-app --example with-expo with-expo-typescript-app
```

Comment:
@arvigeus it should be `--example with-expo-typescript`, with this change it will continue to use the wrong example.

Reply:
The other examples use `-app` suffix, so it should be that way.

Reply:
What I mean is that it should be like this:

```bash
npx create-next-app --example with-expo-typescript with-expo-typescript-app
```

Replace `--example with-expo` with `--example with-expo-typescript`, as it currently stands it will continue to use the `with-expo` example.

Reply:
Grrr, this is already merged, I will have to create another PR. Good catch, @lfades !